src/python/ddapp/ik.py

```diff
@@ -369,3 +369,61 @@ def tick(self):
 
         if self.handleAsyncTasks() > 0:
             return
+            
+    def runMultiRRT(self, grHand, qStart, xGoal, objectGrasped):
+        
+        commands = ['info = test_multiRRT_reach(r, s, \'%s\', %s, %s, %d);\n'%(grHand, qStart, xGoal, objectGrasped)]
+        self.comm.sendCommands(commands)
+
+        info = self.comm.getFloatArray('info')[0]
+        
+        return info
+        
+    def addAffordanceToHand(self, handLink, affordance, q, name):
+        affStr = ''
+        desc = affordance.getDescription()
+        if desc['classname'] == 'BoxAffordanceItem':
+            affStr += 'RigidBodyBox(%s, %s, quat2rpy(%s));,'%(desc['Dimensions'], list(desc['pose'][0]), list(desc['pose'][1]))
+        elif desc['classname'] == 'CylinderAffordanceItem':
```

Comment:
this function should probably raise an exception for unsupported affordance types